packages/rum/test/viewCollection.spec.ts

```diff
@@ -66,10 +66,25 @@ function mockHistory(location: Partial<Location>) {
     const url = `http://localhost${pathname}`
     location.pathname = getPathName(url)
     location.search = getSearch(url)
-    location.hash = getHash(url)
+    location.hash = getHash(url) || ''
   })
 }
 
+function mockHash(location: Partial<Location>) {
+  // Reset previous tests value
+  window.location.hash = ''
```

Comment:
what about moving that to the cleanup method?
it would avoid conflict with other/future tests